plugin/src/App/init.lua

```diff
@@ -65,6 +69,8 @@ function App:startSession(host, port, sessionOptions)
 				projectName = details,
 				address = address,
 			})
+
+			Log.warn(string.format("Connected to session '%s' at %s", details, address))
```

Comment:
These warnings are helpful for users using the keybinds, but they probably should not be warnings.

I want to add a notification system for Rojo at some point. In the meantime, are the icons from #538 enough of an indicator? Can we remove these warnings?

Reply:
Removing them leads to zero indication for me, even with #538 because I don't leave the plugin ribbon open.

I could change them to Log.info instead of Log.warn, if that's better.

A notification system that puts popups into CoreGui would be nice as that would show regardless of widget or ribbon state.

Reply:
Dropping them to `Log.info` would be fine.

Agreed on notifications, I think that's where we would want to go eventually, but it's out of scope for this PR.